tests/e2e/default/db/ongoing-replication.wdio-spec.js

```diff
@@ -169,16 +167,13 @@ describe('ongoing replication', function() {
   });
 
   it('should download settings updates', async () => {
-    await browser.throttle('offline');
+    await commonPage.sync();
     await utils.updateSettings({ test: true }, 'api');
-    await browser.throttle('online');
     await commonPage.sync(true);
     const [settings] = await chtDbUtils.getDocs(['settings']);
     expect(settings.settings.test).to.equal(true);
 
-    await browser.throttle('offline');
     await utils.revertSettings(true);
-    await browser.throttle('online');
```

Comment:
These tests frequently failed because the update modal showed up while we were clicking around in the hamburger menu. I thought I could control it by forcefully preventing the browser from connecting, or by syncing before the test began, but it didn't help. Setting the browser as offline doesn't also set the service worker instance, which is separate, to use the same setting. 
The time when the modal appeared was unpredictable, and it seemed like I would have to check for its existence before every click.